src/scratchtocatrobat/converter/converter.py

```diff
@@ -62,7 +62,6 @@
 CATROBAT_DEFAULT_SCENE_NAME = "Scene 1"
 UNSUPPORTED_SCRATCH_BLOCK_NOTE_MESSAGE_PREFIX_TEMPLATE = "Missing brick for Scratch identifier: [{}]"
 UNSUPPORTED_SCRATCH_FORMULA_BLOCK_NOTE_MESSAGE_PREFIX = "Missing formula element in brick: [{}] for Scratch identifier: [{}]"
-XML_CHARACTERS_TO_BE_REPLACED_MAPPING = {'\'': '', '<': 'lessThan', '>': 'greaterThan', '&': 'AND' }
```

Comment:
I guess this mapping comes from a time when Pocket Code couldn't parse XML correctly.
Please have a look and test this before approving or merging.